### ISSUE TITLE
[Informative] Observable changes because of Unicode 13

### ISSUE THREAD

**Issue by mathiasbynens** [tc39/ecma262#1897]
Output of [`node compare.js`]( https://gist.github.com/mathiasbynens/c08f45efdcdb9389adf3da57e16683de):

```
$ node compare.js 
Comparing Unicode v13.0.0 to Unicode v12.1.0…
Binary_Property: 0 new values
General_Category: 0 new values
Script: 4 new values
- Chorasmian (Chrs)
- Dives_Akuru (Diak)
- Khitan_Small_Script (Kits)
- Yezidi (Yezi)
Script_Extensions: 4 new values
- Chorasmian (Chrs)
- Dives_Akuru (Diak)
- Khitan_Small_Script (Kits)
- Yezidi (Yezi)
Binary_Property/ID_Start: 0 removals
Binary_Property/ID_Start: 5,598 new code points
Binary_Property/ID_Continue: 0 removals
Binary_Property/ID_Continue: 5,645 new code points
General_Category/Space_Separator: 0 removals
General_Category/Space_Separator: 0 new code points
Case_Folding=S: 0 removals
Case_Folding=S: 0 new code points
Case_Folding=F: 0 removals
Case_Folding=F: 0 new code points
```

TL;DR identifier symbols are expanded and some new `Script`/`Script_Extensions` properties and aliases are supported in RegExp Unicode property escapes (#1896).

**Comment by ljharb** [comment]
Where could I see a list of the new characters that will be available for use in identifiers?

**Comment by mathiasbynens** [comment]
@ljharb https://gist.github.com/mathiasbynens/c08f45efdcdb9389adf3da57e16683de#file-compare-js-L50

**Comment by mathiasbynens** [comment]
You’d want to compare

```js
require('unicode-13.0.0/Binary_Property/ID_Start/code-points.js');
```

against

```js
require('unicode-12.1.0/Binary_Property/ID_Start/code-points.js');
```

Same for `ID_Continue` instead of `ID_Start`.

**Comment by mathiasbynens** [comment]
> Where could I see a list of the new characters that will be available for use in identifiers?

https://github.com/mathiasbynens/caniunicode now answers this question. It generates Test262 tests for each Unicode version that introduces new identifier characters.

**Comment by mathiasbynens** [comment]
Closing since this was purely filed as an informative issue.
